package-lock.json

```diff
@@ -2012,9 +2012,9 @@
       }
     },
     "commander": {
-      "version": "2.20.0",
-      "resolved": "https://registry.npmjs.org/commander/-/commander-2.20.0.tgz",
-      "integrity": "sha512-7j2y+40w61zy6YC2iRNpUe/NwhNyoXrYpHMrSunaMG64nRnaf96zO/KMQR4OyN/UnE5KLyEBnKHd4aG3rskjpQ=="
+      "version": "4.0.1",
```

Comment:
Did they go straight from 2.x to 4.x?  I want to make sure we upgrade properly:
https://github.com/tj/commander.js/releases

Reply:
They had a `3` in the middle. Nothing in that changelog scared me when I looked at it (the biggest thing was officially dropping Node 6, which we've already done).

The arguments thing might of `4.0` might be something worth looking into further - it'd be related to the block that I (and Michael) commented on.

Is there something specific that concerns you?

Reply:
No just double-checking we know what `3` did.